_posts/2023-02-05-ten-years-of-scala-js.md

```diff
@@ -0,0 +1,91 @@
+---
+category: blog-detail
+post-type: blog
+by: Sébastien Doeraene
+title: "10 years of Scala.js"
+---
+
+10 years ago, on February 5, 2013, we made the [first commit to Scala.js](https://github.com/scala-js/scala-js/commit/9ad7627c2418e5d345375705ca087a60e3aa2c22).
+When it started, compiling Scala to JavaScript was a dream.
+Today, it has become everyday reality.
+[Scala.js](https://www.scala-js.org/) is a mature technology with a rock-solid core, a vibrant ecosystem of dedicated libraries, and an enthusiastic community of users and contributors.
+
+As we celebrate its 10-year anniversary, let us look back at what made it possible.
+
+## How it started
+
+When we first started developing Scala.js, it was a one-person project at [LAMP, EPFL](https://www.epfl.ch/labs/lamp/)--the research lab led by Martin Odersky.
+
+Generating JavaScript from Scala was not a new idea.
+There had already been several attempts at doing so, both at LAMP ([Scala+GWT](https://scalagwt.github.io/), [JS Scala](https://infoscience.epfl.ch/record/179888/)) and in the community ([s2js](https://github.com/alvaroc1/s2js), [JScala](https://github.com/nau/jscala)).
+It felt like Mission Impossible at the time, which was exactly the motivation we needed to get started.
+
+Scala.js was met with enthusiasm when it was first shown off at Scala Days in June 2013--despite a 16 MB "Hello world" output.
+A small group of early enthusiasts gathered on a mailing list and started exchanging their pet projects, providing valuable early feedback to the project.
+The one-person project quickly grew into a two-person core team along with a few dozens of enthusiasts.
```

Comment:
```suggestion
The one-person project quickly grew into a two-person core team along with a few dozen of enthusiasts.
```

Reply:
it's even more idiomatic if you drop the "of": "a few dozen enthusiasts"